i2c/ti/dacx578.go

```diff
@@ -0,0 +1,102 @@
+// Package ti contains drivers for IC's produced by Texas Instruments.
+package ti
+
+import (
+	"fmt"
+	"math"
+
+	"golang.org/x/exp/io/i2c"
+)
+
+const (
+	// wuChannel simultaneously writes data to the selected
+	// register while updating selected DAC register.
```

Comment:
I find this comment a bit confusing and I think it isn't completely true. `wuChannel` doesn't write itself, but it's the command used for writing.

I suggest to rephrase it to something like this:

``` go
// cmd is the command used to write to DAC input register channeln, 
// and update DAC register channel n. See table 6 of the datasheet.
cmd = 0x30
```